support-workers/src/main/scala/com/gu/zuora/productHandlers/ZuoraDigitalSubscriptionDirectHandler.scala

```diff
@@ -41,7 +41,7 @@ class ZuoraDigitalSubscriptionDirectHandler(
       state.product,
       state.paymentMethod,
       paymentSchedule,
-      state.promoCode,
+      state.appliedPromotion.map(_.promoCode),
```

Comment:
Later states in the support-workers step function still have the `promoCode` field. This is because we have already done the necessary validation by this point so we don't need the full `appliedPromotion` fields any more.